packages/client/components/KudosReceivedNotification.tsx

```diff
@@ -0,0 +1,59 @@
+import graphql from 'babel-plugin-relay/macro'
+import {Link} from 'react-router-dom'
+import React, {useEffect} from 'react'
+import {useFragment} from 'react-relay'
+import {KudosReceivedNotification_notification$key} from '~/__generated__/KudosReceivedNotification_notification.graphql'
+import NotificationTemplate from './NotificationTemplate'
+import useAtmosphere from '../hooks/useAtmosphere'
+import SendClientSideEvent from '../utils/SendClientSideEvent'
+import getReactji from '~/utils/getReactji'
+
+interface Props {
+  notification: KudosReceivedNotification_notification$key
+}
+
+const KudosReceivedNotification = (props: Props) => {
+  const atmosphere = useAtmosphere()
+  const {notification: notificationRef} = props
+  const notification = useFragment(
+    graphql`
+      fragment KudosReceivedNotification_notification on NotifyKudosReceived {
+        ...NotificationTemplate_notification
+        id
+        type
+        name
+        picture
+        meetingName
+        meetingId
+        emoji
+      }
+    `,
+    notificationRef
+  )
+  const {type, name, picture, meetingName, emoji, meetingId} = notification
+
+  const {unicode} = getReactji(emoji)
+
+  useEffect(() => {
+    SendClientSideEvent(atmosphere, 'Notification Viewed', {
```

Comment:
+1 This seems to send a client event for every kudos notification in the dropdown every time the user opens the notification dropdown menu.

Would it be more helpful to only send the client event if it's unread?

Reply:
I made it similar to other notifications. @tghanken what do you think?

Reply:
I think this is fine as is.  It could be helpful to add "isRead" ads a boolean parameter if it's easy!

Reply:
we have notification status, CLICKED, READ etc. I can add it